doctoshotgun.py

```diff
@@ -617,6 +617,16 @@ def setup_loggers(self, level):
         logging.root.setLevel(level)
         logging.root.addHandler(self.create_default_logger())
 
+    def try_to_book_or_sleep(self, docto, center, vaccine_list, start_date, end_date, only_second, only_third, dry_run=False):
```

Comment:
To me the method name is a bit misleading. "or sleep" in fact describes the internal logic of a lazy loading `try_to_book || sleep` but from the sound of it I would expect "or sleep" is related to the return value. Maybe someone comes up with a better name?

Reply:
Yeah your right, i guess we could just use `try_to_book`? I mean it's just that and in fact it does not matter if there is a `sleep` inside or not. What do you think?